main.tf

```diff
@@ -26,6 +26,8 @@ resource "aws_elasticache_subnet_group" "default" {
   name        = module.labels.id
   subnet_ids  = var.subnet_ids
   description = var.description
+
+  tags = merge(var.tags, module.labels.tags)
```

Comment:
we have a variable in labels called extra_lables for extra labels please use this